packages/edit-post/src/components/welcome-guide/images.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const CanvasImage = ( props ) => (
+	<img
+		alt=""
```

Comment:
Shouldn't those `alt` attributes be filled in for those who use screen readers?

Reply:
From https://github.com/WordPress/gutenberg/pull/18041#issuecomment-556508056:

> I don't think the images need an alt description, because the accompanying text does a good job of describing each page/topic. That being said, they should have an empty alt = "" so that they're ignored by AT.